src/mscorlib/shared/System.Private.CoreLib.Shared.sources

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TargetsWindows Condition="'$(TargetsWindows)' != 'true'">false</TargetsWindows>
+    <TargetsUnix Condition="'$(TargetsUnix)' != 'true'">false</TargetsUnix>
+    <TargetsOSX Condition="'$(TargetsOSX)' != 'true'">false</TargetsOSX>
+  </PropertyGroup>
+  <ItemGroup>
+    <MscorlibSources Include="$(MSBuildThisFileDirectory)\System\Buffers\*.cs"/>
+    <MscorlibSources Include="$(MSBuildThisFileDirectory)\System\IO\Error.cs"/>
+    <MscorlibSources Include="$(MSBuildThisFileDirectory)\System\IO\FileStream.cs"/>
+    <MscorlibSources Include="$(MSBuildThisFileDirectory)\System\IO\Path.cs"/>
+    <MscorlibSources Include="$(MSBuildThisFileDirectory)\System\IO\PathInternal.cs"/>
+    <!--<MscorlibSources Include="$(MSBuildThisFileDirectory)\Microsoft\Win32\SafeHandles\SafeThreadPoolIOHandle.cs"/>-->
```

Comment:
This file should be deleted instead since it is used in CoreCLR. Looks like it was added to CoreCLR by accident.